packages/aws-cdk/lib/settings.ts

```diff
@@ -18,23 +19,28 @@ export class Configuration {
   public readonly userConfig = new Settings();
   public readonly projectConfig = new Settings();
 
-  constructor(commandLineArguments?: Settings) {
+  constructor(commandLineArguments?: Settings | yargs.Arguments) {
```

Comment:
Don't think this needs to be backwards compatible.

Reply:
That's true. I'll get rid of the redundant crap when I have a few cycles to spare.